catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -151,6 +151,14 @@ const SearchLayout: FunctionComponent = ({ children }) => {
             return false;
           }}
         />
+        <h1 className="visually-hidden">
```

Comment:
Tbh this content is subject to change, I just wanted to move it out of the individual files as we were rendering a different component should the query string be empty.